bin/rose-check-software

```diff
@@ -298,7 +298,7 @@ def docs(check=check):
     ret = check_all('Documentation Builder', [
         check('python', (2, 7), (3,), version_template=r'Python (.*)',
                outfile=2),
-        check('dot', command_template='-V',
+        check('graphviz-dot <dot>', command_template='-V',
```

Comment:
One other (minor) thing: the space (which might be necessary?) between 'graphviz' and '<dot>' gives an extra space between the text and the delimited dots in the output, i.e:
```
python (2.7+, <3) ...................................... not ok (2.6.6 < 2.7)
graphviz-dot  .......................................... ok (2.26.0)
tex .................................................... ok (3.141592)
```

Reply:
Sorted.